lib/Instance/CustomVariable.js

```diff
@@ -270,6 +270,15 @@ export default class InstanceCustomVariable extends CoreBase {
 		}
 	}
 
+	getValue(name) {
+		if (this.custom_variables[name]) {
+			this.logger.silly(`Get value "${name}"`)
+			return this.custom_variables[name]
+		} else {
+			return 'Unknown name'
```

Comment:
It would be better to return `undefined` here rather than a value. That will let the caller decide what to do when it doesnt exist, instead of thinking it has a value of `Unknown name`